src/account/useEmailSignIn.ts

```diff
@@ -10,7 +10,7 @@ type TRequest = {
 }
 /**
  * Create email session
- * {@link https://appwrite.io/docs/client/account?sdk=web-default#accountCreateEmailSession | create email session documentation}
+ * @link [appwrite create email session documentation](https://appwrite.io/docs/client/account?sdk=web-default#accountCreateEmailSession) 
```

Comment:
This is close. I should have been clearer, my apologies. It should just say `Appwrite Documentation` between the square brackets.